packages/manager/modules/pci/src/projects/project/instances/instances.html

```diff
@@ -83,60 +83,60 @@ <h1 data-translate="pci_projects_project_instances_title"></h1>
             data-oui-popover-template="pci/projects/project/instances/help/backup.html"
             data-oui-popover-scope="$ctrl">
             <oui-action-menu-item
-                data-text="{{ 'pci_projects_project_instances_view_label' | translate }}"
+                data-text="{{:: 'pci_projects_project_instances_view_label' | translate }}"
                 data-on-click="$ctrl.viewInstance($row)">
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{ 'pci_projects_project_instances_edit_label' | translate }}"
+                data-text="{{:: 'pci_projects_project_instances_edit_label' | translate }}"
                 data-on-click="$ctrl.editInstance($row)"
                 data-ng-if="$row.statusGroup === 'ACTIVE'">
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{ 'pci_projects_project_instances_activeMonthlyBilling_label' | translate }}"
+                data-text="{{:: 'pci_projects_project_instances_activeMonthlyBilling_label' | translate }}"
                 data-on-click="$ctrl.enableMonthlyBillingInstance($row)"
                 data-ng-if="$row.status === 'ACTIVE' && !$row.isMonthlyBillingActivated()">
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{ 'pci_projects_project_instances_createBackup_label' | translate }}"
+                data-text="{{:: 'pci_projects_project_instances_createBackup_label' | translate }}"
                 data-on-click="$ctrl.createBackupInstance($row)"
                 data-ng-if="$row.statusGroup === 'ACTIVE'">
             </oui-action-menu-item>
             <oui-action-menu-divider
                 data-ng-if="$row.statusGroup === 'ACTIVE' || $row.status === 'RESCUE'">
             </oui-action-menu-divider>
             <oui-action-menu-item
-                data-text="{{ 'pci_projects_project_instances_rescue_label' | translate }}"
+                data-text="{{:: 'pci_projects_project_instances_rescue_label' | translate }}"
                 data-on-click="$ctrl.startRescueInstance($row)"
                 data-ng-if="$row.statusGroup === 'ACTIVE'">
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{ 'pci_projects_project_instances_rescueEnd_label' | translate }}"
+                data-text="{{:: 'pci_projects_project_instances_rescueEnd_label' | translate }}"
                 data-on-click="$ctrl.endRescueInstance($row)"
                 data-ng-if="$row.status === 'RESCUE'">
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-text="{{ 'pci_projects_project_instances_softReboot_label' | translate }}"
+                data-text="{{:: 'pci_projects_project_instances_softReboot_label' | translate }}"
```

Comment:
We are not supposed to use the `text` attribute, it is deprecated 

https://ovh-ux.github.io/ovh-ui-kit/?path=/story/components-action-menu--simple
```suggestion
                >
                <span data-translation="pci_projects_project_instances_softReboot_label"></span>
```